gym_cap/envs/create_map.py

```diff
@@ -85,46 +85,65 @@ def gen_map(name, dim=20, in_seed=None, rand_zones=False, np_random=None,
         return new_map, static_map
     
     @staticmethod
-    def dest_map(new_map,\
-                 map_obj=[NUM_BLUE, NUM_UAV, NUM_RED, NUM_UAV, NUM_GRAY]):
+    def dest_map(new_map):
         """
         Method
-            Outputs static_map when new_map is given as input
-            Addtionally the number of agents can also be changed
+            Outputs static_map when new_map is given as input.
+            Addtionally the number of agents will also be
+            counted
         
         Parameters
         ----------
-        agent_map        : numpy array
+        new_map        : numpy array
             new_map
-        map_obj     : list
-            The necessary elements to build the map
-            0   : blue UGV
-            1   : blue UAV
-            2   : red UGV
-            3   : red UAV
-            4   : gray units
+        The necessary elements:
+            ugv_1   : blue UGV
+            uav_1   : blue UAV
+            ugv_2   : red UGV
+            uav_2   : red UAV
+            gray    : gray units
+            
         """
         orig_map = np.copy(new_map)
```

Comment:
Static map should not contain moving element. There needs to be a step that replace the element to background. Only obstacle, background, and flag should be in static map.